src/libraries/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -181,7 +181,7 @@ public sealed partial class EventHandlerList : System.IDisposable
         public void RemoveHandler(object key, System.Delegate? value) { }
     }
     [System.ComponentModel.DesignerAttribute("System.ComponentModel.Design.ComponentDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
-    [System.ComponentModel.DesignerAttribute("System.Windows.Forms.Design.ComponentDocumentDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+    [System.ComponentModel.DesignerAttribute("System.Windows.Forms.Design.ComponentDocumentDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", typeof(System.ComponentModel.Design.IRootDesigner))]
```

Comment:
You'll need the fully qualified name for this one.  It lives in TypeConverter on core.  It lived in System on .NETFramework.  Let's use the .NETFramework identity.

Reply:
Good catch. I fixed this and also fixed another instance of this in another DesignerAttribute that used this designer.